TileFormats/FeatureTable/schema/featureTable.schema.json

```diff
@@ -0,0 +1,91 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema#",
+    "id" : "featureTable.schema.json",
+    "title" : "Feature Table",
+    "type" : "object",
+    "description" : "A set of semantics containing per-tile and per-feature values defining the position and appearance properties for features in a tile.",
+    "patternProperties" : {
+        ".*" : {
+            "properties" : {
+                "byteOffset" : {
+                    "type" : "integer",
+                    "minimum" : 0
+                }
+            }
+        }
+    },
+    "definitions" : {
+        "featureReference" : {
+            "type" : "object",
+            "required" : ["byteOffset"]
+        },
+        "featureArray" : {
+            "type" : "array",
+            "items" : {
+                "type" : "number"
+            }
+        },
+        "feature" : {
+            "oneOf" : [
+                {
+                    "$ref" : "#/definitions/featureReference"
+                },{
+                    "$ref" : "#/definitions/featureArray"
+                }
+            ]
+        },
+        "featureGlobalScalar" : {
```

Comment:
It is a general definition for scalar values. It gets applied to `INSTANCES_LENGTH` and `POINTS_LENGTH`. It says that a single scalar can be stored via reference in the binary, can be an array with only one element, or can be an integer >= 0.